content/05.main-text.md

```diff
@@ -1,9 +1,63 @@
 ## Introduction {.page_break_before}
 
+A *network* (also known as a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics))) is a conceptual representation of a group of entities — called *nodes* — and the relationships between them — called *edges*.
+Typically, a network has only one type of node and one type of edge.
+But in many cases, it is necessary to be able to distinguish between different types of entities and relationships.
+
+### Hetnets
+
+A *hetnet* (short for **het**erogeneous information **net**work [@doi:10.15363/thinklab.d104]) is a network where nodes and edges have type.
+The ability to differentiate between different types of entities and relationships allows a hetnet to accurately describe more complex data.
+Hetnets are particularly useful in biomedicine, where it is important to capture the conceptual distinctions between various concepts,
+such as genes and diseases, or upregulation and binding.
+
+The types of nodes and edges in a hetnet are defined by a schema, referred to as a metagraph.
+The metagraph consists of metanodes (types of nodes) and metaedges (types of edges).
+Note that the prefix *meta* is used to refer to type (e.g. compound),
+as opposed to a specific node/edge/path itself (e.g. acetaminophen).
```

Comment:
I've found this natural - I wonder if you want to add a metaedge mention as well? For me, that was harder to think about at first than nodes having different types. Especially since there can be multiple different metaedges between the same metanodes.